docs/includes/schema-builder/astronauts_migration.php

```diff
@@ -0,0 +1,30 @@
+<?php
+
+declare(strict_types=1);
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Support\Facades\Schema;
+use MongoDB\Laravel\Schema\Blueprint;
+
+return new class extends Migration
+{
+    protected $connect = 'mongodb';
```

Comment:
This property name is incorrect. Same for other examples.
```suggestion
    protected $connection = 'mongodb';
```

https://github.com/laravel/framework/blob/11.x/src/Illuminate/Database/Migrations/Migration.php#L12